recipes/xz/xz.inc

```diff
@@ -0,0 +1,49 @@
+DESCRIPTION = "XZ Utils is free general-purpose data compression software with a high compression ratio, the successor to LZMA Utils"
+HOMEPAGE = "http://tukaani.org/xz/"
+
+COMPATIBLE_HOST_ARCHS = ".*linux"
+RECIPE_TYPES = "machine native"
+
+# The COPYING file starts
+#
+#   Different licenses apply to different files in this package. Here
+#   is a rough summary of which licenses apply to which parts of this
+#   package (but check the individual files to be sure!):
+#
+# Boiling that down to a single license tag is not easy. Most files
+# are public domain, the remaining are under some GPL variant.
+
+LICENSE = "GPLv2"
```

Comment:
GPL-2.0

you will need to add something additional to reflect what is stated above. Users of this recipe will look in bom.txt for what licenses is used